Makefile

```
@@ -48,11 +48,30 @@ GO_BUILD_LDFLAGS_S := true
 ifeq ($(KEEP_SYMBOLS),1)
 	GO_BUILD_LDFLAGS_S = false
 endif
-GO_BUILD_LDFLAGS := -ldflags="-s=$(GO_BUILD_LDFLAGS_S) -w -X $(PACKAGE)/pkg/version.Version=$(VERSION)"
+
+DEBUG ?=
+GO_BUILD_GCFLAGS ?=
+GO_BUILD_LDFLAGS_FLAGS ?=
+ifeq ($(DEBUG),1)
+	# Disable optimizations and inlining to make debugging easier.
+	GO_BUILD_GCFLAGS = -gcflags="all=-N -l"
+	# Keep the symbol table
+	GO_BUILD_LDFLAGS_FLAGS += -s=false
```

```suggestion
	KEEP_SYMBOLS = 1
```

Also move the `ifeq ($(KEEP_SYMBOLS),1)` section to make this effective

---

`-ldflags`へ渡すパラメータの順番を維持するために`KEEP_DWARF`も追加しました。